server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@jordicrosmompart you shouldn't need to duplicate this file here, can use the same file used for the char driver.